package.json

```diff
@@ -57,13 +53,13 @@
   "dependencies": {},
   "devDependencies": {
     "benchmark": "^2.1.0",
-    "browserify": "^13.0.0",
-    "browserify-istanbul": "^1.0.0",
-    "coveralls": "2.11.9",
-    "eslint": "^2.9.0",
-    "eslint-config-strict": "^8.5.0",
-    "eslint-plugin-filenames": "^0.2.0",
-    "ghooks": "^1.0.1",
+    "browserify": "^14.3.0",
+    "browserify-istanbul": "^2.0.0",
+    "coveralls": "2.13.0",
+    "eslint": "^3.19.0",
+    "eslint-config-strict": "^13.0.0",
+    "eslint-plugin-filenames": "^1.1.0",
+    "husky": "^0.13.3",
```

Comment:
Yeah sorry - I should have mentioned; [ghooks is now deprecated](https://github.com/gtramontina/ghooks#deprecated) in favour of husky.